test/house_test.rb

```diff
@@ -0,0 +1,110 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class RoomTest < Minitest::Test
+  def test_it_exists
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_instance_of House, house
+  end
+
+  def test_it_has_a_price_and_address
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_equal 400000, house.price
+    assert_equal "123 sugar lane",house.address
+  end
+
+  def test_rooms_can_be_added
+    house = House.new("$400000", "123 sugar lane")
+    room_1 = Room.new(:bedroom, 10, '13')
+    room_2 = Room.new(:bedroom, 11, '15')
+
+    assert_equal [], house.rooms
+
+    house.add_room(room_1)
+    house.add_room(room_2)
+    assert_equal [room_1, room_2], house.rooms
+  end
+
+  def test_it_can_be_above_or_below_market_average
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_equal false, house.above_market_average?
```

Comment:
if you have a method that can return multiple possible outcomes, test both.  I'd expect to see another house here that is above market average, and an assertion to go with that.

Reply:
I did add a test for the additional area room method I made, but I missed this one!  Productive struggle!  :)